### PR TITLE
`Communication`: Update screencasts in documentation

### PR DESCRIPTION
**Pull request by egekurt123**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/development-process.html#naming-conventions-for-github-pull-requests).

### Description

This PR updates the screencasts in the document of communication module, ensuring they accurately represent the current interface.

#### For easier review process

Original Page: https://docs.artemis.cit.tum.de/user/communication.html
Changed Page: https://artemis-platform--9176.org.readthedocs.build/en/9176/user/communication.html

#### Review
- [x]  Review 1
- [x]  Review 2
- [x]  Review 3
- [x]  Review 4

## Summary by CodeRabbit

- **New Features**
	- Updated embedded video links in the user communication documentation to ensure access to the latest instructional content.

- **Style**
	- Adjusted the height of several video displays from 450 to 350 for improved integration within the documentation layout.

### CODE REVIEW

**Review by JohannesStoehr — DISMISSED** [review]
Looks good to me

**Review by pzdr7 — DISMISSED** [review]
LGTM👍

**Review by coolchock — DISMISSED** [review]

**Review by edkaya — DISMISSED** [review]

File: docs/user/communication.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ This video provides an overview of the course-wide channel types existing in a c
 
 .. raw:: html
 
-    <iframe src="https://live.rbg.tum.de/w/artemisintro/42164?video_only=1&t=0" allowfullscreen="1" frameborder="0" width="600" height="450">
+    <iframe src="https://live.rbg.tum.de/w/artemisintro/47622?video_only=1&t=0" allowfullscreen="1" frameborder="0" width="600" height="450">
```
Comment (rabeatwork) [inline docs/user/communication.rst:37]: Should we change the height here, too? (height="350")?

**Review by sarpsahinalp — APPROVED** [review]
Page lgtm

**Review by asliayk — APPROVED** [review]
lgtm

**Review by pzdr7 — APPROVED** [review]
Reapprove

**Pull request merged**